app/tts.py

```
@@ -0,0 +1,38 @@
+    mp3_path: str | Path,
+    title: str,
+    album: str = "AIニュースポッドキャスト",
+    artist: str = "AutoNarrator",
+    comment: str | None = None,
+) -> None:
+    path = Path(mp3_path)
+    if not path.exists():
+        return
+    try:
+        audio = ID3(str(path))
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)